danger-klaxit/lib/danger_plugin.rb

```diff
@@ -59,6 +59,7 @@ def fail_for_bad_commits
   def run_brakeman_scanner
     configs = { app_path: "." }
```

Comment:
A tiny change that will help us later!
```suggestion
  def run_brakeman_scanner(configs = { app_path: "." })
```